### PR TITLE
Migration/l10n br sale product rebased

### PR DESCRIPTION
**Pull request by rvalyi**
should replace https://github.com/OCA/l10n-brazil/pull/146 where I screwed a rebase when fixing a conflict. Now the rebase should be clean.

### CODE REVIEW

**Comment by rvalyi** [comment]
rebased so wkf issue should be fixed. Let's see...

**Comment by rvalyi** [comment]
green. Ok for merge @mileo @danimaribeiro ?

File: l10n_br_sale_product/models/sale.py [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+            self.product_id.id,
+            self.order_id.partner_invoice_id.id,
+            fiscal_position=self.fiscal_position,
+            insurance_value=self.insurance_value,
+            freight_value=self.freight_value,
+            other_costs_value=self.other_costs_value)
+
+        self.price_subtotal = (self.order_id.pricelist_id
+                               .currency_id.round(taxes['total']))
+        self.price_gross = self.price_unit * self.product_uom_qty
```
Comment (mileo) [inline l10n_br_sale_product/models/sale.py]: Please use the l10n_br_sale price_gross function

File: l10n_br_sale_product/res_company.py [hunk 2]
```diff
@@ -1,32 +0,0 @@
-# -*- encoding: utf-8 -*-
-###############################################################################
-#                                                                             #
-# Copyright (C) 2013  Luis Felipe Miléo - luisfelipe@mileo.co                 #
```
Comment (mileo) [inline l10n_br_sale_product/res_company.py:4]: Can you put me in the contributors?

**Comment by mileo** [comment]
@mstuttgart @sadamo @fernandomr 
Please some one review for me. I can not do test it in the next 48 hours.

File: l10n_br_sale_product/models/sale.py (hunk 1 above)
Comment (rvalyi) [inline l10n_br_sale_product/models/sale.py]: right. We forgot to propagate the l10n_br_sale refactor you suggested indeed.

File: l10n_br_sale_product/res_company.py (hunk 2 above)
Comment (rvalyi) [inline l10n_br_sale_product/res_company.py:4]: yup

**Comment by mileo** [comment]
Some fixes: https://github.com/akretion/l10n-brazil/pull/8

**Comment by mileo** [comment]
With https://github.com/akretion/l10n-brazil/pull/8 :+1:

**Comment by rvalyi** [comment]
:+1: LGTM

**Comment by mileo** [comment]
:+1: LGTM

**Comment by fernandomr** [comment]
:+1:  LGTM!

**Pull request merged**